jvm/src/main/java/com/microsoft/bond/protocol/UnsignedHelper.java

```diff
@@ -0,0 +1,40 @@
+package com.microsoft.bond.protocol;
+
+import java.math.BigInteger;
+
+public class UnsignedHelper {
+    private final static BigInteger TWO_TO_65;
+    private final static BigInteger LONG_MAX_PLUS_ONE;
+    static {
```

Comment:
Style: I think it's easier to read if you initialize the field on one line without a static block.